cmd/erasure-sets.go

```diff
@@ -341,7 +337,7 @@ func newErasureSets(ctx context.Context, endpoints Endpoints, storageDisks []Sto
 
 	listTolerancePerSet := 3
 	// By default this is off
-	if env.Get("MINIO_API_LIST_STRICT_QUORUM", config.EnableOff) == config.EnableOn {
+	if env.Get("MINIO_API_LIST_STRICT_QUORUM", config.EnableOn) == config.EnableOn {
```

Comment:
@harshavardhana It now is default ON. Is this intentional?

Reply:
Yes for the time being @klauspost we are seeing a lot of stale listing issues.